js/App.js

```diff
@@ -1,13 +1,17 @@
 const React = require("react");
 const Title = require("./components/Title");
 const Description = require("./components/Description");
+const ColorTag = require("./components/ColorTag");
 
 class App extends React.Component {
 	render() {
+		const libName = "React";
+
 		return (
 			<div className="app">
-				<Title>Test Title</Title>
-				<Description library="will.js"/>
+				<Title>Yo this be a test</Title>
+				<Description library="React"/>
+				<ColorTag color = "purple" size = "4"> Rendering this color text</ColorTag>
```

Comment:
Much like with my Title example, just because you're _passing_ the props, doesn't mean they're being used. See line 6 of your ColorTag component. It will still be purple, even if you pass orange.